test/e2e/generate_systemd_test.go

```diff
@@ -228,7 +228,7 @@ var _ = Describe("Podman generate systemd", func() {
 		n.WaitWithDefaultTimeout()
 		Expect(n.ExitCode()).To(Equal(0))
 
-		session := podmanTest.Podman([]string{"generate", "systemd", "--timeout", "42", "--name", "--new", "foo"})
+		session := podmanTest.Podman([]string{"generate", "systemd", "--time", "42", "--name", "--new", "foo"})
```

Comment:
Can we leave one `--timeout` to make sure we're not regressing on it? Maybe there's one left (I didn't checkout the PR to check).

Reply:
Yes I left some --timeout(s)

Reply:
Awesome!

Reply:
```
cd test
grep -- --timeout . -r
./e2e/stop_test.go:	It("podman stop container --timeout", func() {
./e2e/stop_test.go:		session = podmanTest.Podman([]string{"stop", "--timeout", "1", "test5"})
./e2e/libpod_suite_remoteclient_test.go:	args := []string{"varlink", "--timeout", "0", varlinkEndpoint}
./e2e/generate_systemd_test.go:		session := podmanTest.Podman([]string{"generate", "systemd", "--timeout", "42", "--name", "--new", "foo"})
./apiv2/test-apiv2:    $PODMAN_BIN --root $WORKDIR system service --timeout 15 tcp:127.0.0.1:$PORT \
./endpoint/endpoint.go:	args := []string{"varlink", "--timeout", "0", varlinkEndpoint}
./system/050-stop.bats:    # Because the --time and --timeout options can be wonky, try three
./system/050-stop.bats:    for t_opt in '' '--time=5' '--timeout=5'; do
```